blob/service.go

```diff
@@ -219,6 +219,10 @@ func (s *Service) getByCommitment(
 	)
 
 	for _, row := range namespacedShares {
+		if len(row.Shares) == 0 {
+			return nil, nil, ErrBlobNotFound
```

Comment:
So, to avoid incomplete blob error?

Reply:
Yes. It's also not possible, but I'd like to keep it as safe as possible.

Reply:
Let's add a little comment describing why this if is special and why does it mean that the whole operation is cancelled.